demo/roles/openfoam/tasks/main.yml

```diff
@@ -0,0 +1,44 @@
+- name: Install docker dependencies
+  become_user: root
+  apt: update_cache=yes
+  apt: name=apt-transport-https state=present
+  apt: name=ca-certificates state=present
+
+- name: Add docker key
+  become_user: root
+  apt_key:
+    keyserver=hkp://ha.pool.sks-keyservers.net:80
+    id=58118E89F3A912897C070ADBF76221572C52609D
+
+- name: Add docker repository
+  become_user: root
+  apt_repository:
+    repo: deb https://apt.dockerproject.org/repo ubuntu-trusty main
+    state: present
+
+- name: Add docker to apt
+  become_user: root
+  command: echo "deb https://apt.dockerproject.org/repo ubuntu-trusty main" | tee /etc/apt/sources.list.d/docker.list
+
+- name: Install docker
+  become_user: root
+  apt: update_cache=yes
+  apt: name=docker-engine state=present
+  
```

Comment:
Maybe but that seems overkill for a useless case as OpenFOAM with docker has very little interest.